### PR TITLE
chore(dataarts): dataArts service update API comments

### PR DESCRIPTION
**Pull request by wuzhuanhong**
**What this PR does / why we need it**:

 DataArts Studio service adds Add missing api comments.
 DataArts Studio service fix the wrong api comments.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

1.  DataArts Studio service add missing API comments and fix the wrong api comments.
```

## PR Checklist

* [x] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccSomethingV0_basic'
...
=== RUN   TestAccSomethingV0_basic
--- PASS: TestAccSomethingV0_basic (70.75s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       70.796s
```

### CODE REVIEW

File: huaweicloud/services/dataarts/resource_huaweicloud_dataarts_architecture_business_metric.go [hunk 1]
```diff
@@ -25,6 +25,10 @@ import (
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
 )
 
+// @API DataArtsStudio  POST /v2/{project_id}/design/biz-metrics
```
Comment (ShiChangkuo) [inline huaweicloud/services/dataarts/resource_huaweicloud_dataarts_architecture_business_metric.go]: extra spaces
```suggestion
// @API DataArtsStudio POST /v2/{project_id}/design/biz-metrics
```

Comment (wuzhuanhong) [reply huaweicloud/services/dataarts/resource_huaweicloud_dataarts_architecture_business_metric.go]: fixed

**Pull request merged**